gbasis/evals/electrostatic_potential.py

```diff
@@ -13,23 +13,25 @@ def electrostatic_potential(
     coord_type="spherical",
     threshold_dist=0.0,
 ):
-    """Return the electrostatic potentials of the basis set in the Cartesian form.
+    r"""Return the electrostatic potentials of the basis set in the Cartesian form.
 
     Parameters
     ----------
     basis : list/tuple of GeneralizedContractionShell
         Shells of generalized contractions.
     one_density_matrix : np.ndarray(K_orbs, K_orbs)
         One-electron density matrix in terms of the given basis set.
-        If the basis is transformed using `transform` keyword, then the density matrix is assumed to
-        be expressed with respect to the transformed basis set.
+        If the basis is transformed using `transform` keyword, then the density matrix is assumed
+        to be expressed with respect to the transformed basis set.
     points : np.ndarray(N, 3)
-        Coordinates of the points in space (in atomic units) where the basis functions are
-        evaluated.
-        Rows correspond to the points and columns correspond to the x, y, and z components.
+        Cartesian coordinates of the points in space (in atomic units) where the basis functions
+        are evaluated.
+        Rows correspond to the points and columns correspond to the :math:`x, y, \text{and} z`
```

Comment:
FYI can be replaced with :math:`x, y,` and :math:`z` also. Or :math:`x`, :math:`y`, and :math:`z`